src/pages/Sign/Signup/Signup.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from "react";
+import "./Signup.scss";
+
+class Signup extends Component {
+  constructor() {
+    super();
+    this.state = {
+      emailVal: "",
+      pwVal: "",
+      pwCheckVal: "",
+      nameVal: "",
+      phoneVal: "",
+      addressVal: "",
+      addressDetailVal: "",
+    };
+  }
+
+  handleEmailInput = e => {
+    this.setState({
+      emailVal: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pwVal: e.target.value,
+    });
+  };
+
+  // handlePwCheckInput = e => {
+  //   this.setState({
+  //     pwVal: e.target.value,
+  //   });
+  // };
+
+  handleNameInput = e => {
+    this.setState({
+      nameVal: e.target.value,
+    });
+  };
+
+  handlePhoneInput = e => {
+    this.setState({
+      phoneVal: e.target.value,
+    });
+  };
+
+  handleAddressInput = e => {
+    this.setState({
+      addressVal: e.target.value,
+    });
+  };
```

Comment:
계산된 속성명을 이용해 함수 하나로 작성할 수 있습니다.
단, input 태그에 name 속성을 부여해 주셔야 합니다!

```suggestion
  handleInput = e => {
    this.setState({
      [e.target.name] : e.target.value,
    });
  };
```

Reply:
반영했습니다!